src/modules/sector-industries/sector-industries.controller.test.ts

```diff
@@ -0,0 +1,61 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import Chance from 'chance';
+
+import { GetSectorIndustriesQueryDto } from './dto/get-sector-industries-query.dto';
+import { SectorIndustriesController } from './sector-industries.controller';
+import { SectorIndustriesService } from './sector-industries.service';
+
+const chance = new Chance();
+
+describe('SectorIndustriesController', () => {
+  let sectorIndustriesController: SectorIndustriesController;
+  let sectorIndustriesService: SectorIndustriesService;
+
+  beforeEach(async () => {
```

Comment:
Sorry can this `beforeEach` please be updated to `before` or `beforeAll`.

Reply:
Changed to beforeAll